examples/postgres/main.tf

```diff
@@ -1,27 +1,105 @@
+# Service principal used by Humanitec to provision resources
+
+resource "azuread_application" "humanitec_provisioner" {
```

Comment:
I'm not convinced for creating an account here. I can imagine use cases where user is able to provision resources but he's unable to create new accounts 🤔

Reply:
I think those examples should generally require the least amount of inputs and having a pre-created account, with the right permissions isn't necessarily something people have handy.

If you can't create an account you can always comment the respective blocks, but you at least see which roles etc. are required for this example to run (which is also something we are missing in GCP/AWS packs at the moment :-/).

Reply:
I see your point. But isn't it that you're not allowed to create more privileged user that you are? Anyway, somehow pointing out what permissions you need to provision the resource is definitely missing and we can try with such an approach